pkg/kubectl/describe.go

```diff
@@ -927,6 +935,9 @@ func describeContainers(label string, containers []api.Container, containerStatu
 				}
 				fmt.Fprintf(out, "      %s:\t%s (%s:%s)\n", e.Name, valueFrom, e.ValueFrom.FieldRef.APIVersion, e.ValueFrom.FieldRef.FieldPath)
 			case e.ValueFrom.ResourceFieldRef != nil:
+				if nodeCapacity != nil {
+					fieldpath.MergeContainerResourceLimitsWithCapacity(&container, nodeCapacity)
```

Comment:
I think my main point is that having no limit is not good. Printing "unlimited" would transport this message. Exhausting the node capacity can have any unexpected result.